package.json

```diff
@@ -46,5 +46,8 @@
 		"nx-cloud": "16.4.0",
 		"prettier": "2.8.3",
 		"typescript": "5.2.2"
+	},
+	"dependencies": {
+		"isomorphic-git": "1.25.0"
```

Comment:
What happened here?

Reply:
Whoops, must have run an install command in the wrong folder. That's not intentional